### DISCUSSION TITLE
Suggestion to add n8n to the App store

### DISCUSSION THREAD

**Discussion by Ploonet** [IceWhaleTech/CasaOS#391] [Ideas] [answered]
https://github.com/n8n-io/n8n

n8n is an automation platform, similar to Zapier or Make. What's unique about it is that they offer a self-hosted version. Having it on CasaOS would be great in my opinion.

**Comment by Hedius** [comment]
Hi, you can create a Pull request for the app at: https://github.com/IceWhaleTech/CasaOS-AppStore if you got the app running on casaOS. to speed up the process :)

**Comment by JohnGuan** [comment, marked as answer]
I've converted this idea into an App Request for you.

- #443
